test/fixtures/swagger/param-no-response.yaml

```diff
@@ -1,4 +1,7 @@
 swagger: '2.0'
+info:
+  title: Parameters with no response
```

Comment:
Having no responses would result in this being a invalid Swagger document. @danielgtaylor does it make sense to still have this fixture given it results in an invalid Swagger document?

Given the changes in this PR, it seems unnecessary, but wanted to check before removing.